ehrql/tables/tpp.py

```diff
@@ -552,6 +553,160 @@ class clinical_events_ranges(EventFrame):
+    but there may remain some partial duplicates.
+
+
+    **More Information**
+
+    * [Treatment guidelines](https://www.nice.org.uk/guidance/ta878)
+    * [Draft Data Report](https://docs.google.com/document/d/15o4x9sqHEO-sLm2dTqgm3PyAh72cdgOOmZC4AB3BTNk/) (currently only available to internal staff)
+    """
+
+    age_at_received_date = Series(
+        int,
+        description="""
+            Can occasionally be zero, presumably indicating an unknown or missing value
+            as minimum eligibility age is 12.
+        """,
+    )
+    count = Series(
+        int,
+        description="Number of forms.",
+    )
+    covid_indication = Series(
+        str,
+        description="Treatment setting/indication.",
+        constraints=[
+            Constraint.Categorical(
+                ["non_hospitalised", "hospitalised_with", "hospital_onset"]
+            )
+        ],
+    )
+    current_status = Series(
+        str,
+        description="Status of form/application.",
+        constraints=[
+            Constraint.Categorical(
+                [
+                    "Approved",
+                    "Treatment Complete",
+                    "Treatment Not Started",
+                    "Treatment Stopped",
+                ]
+            )
+        ],
+    )
+    diagnosis = Series(
+        str,
+        description="Always has the value 'Covid-19'.",
+    )
+    form_name = Series(
+        str,
+        description="""
+            Name and version of the patient registration form used to register the
+            treatment.
+        """,
+    )
+    intervention = Series(
+        str,
+        description="""
+            Intervention or therapeutic name. Expected to be one of:
+
+             * Paxlovid
+             * Sotrovimab
+             * Molnupiravir
+             * Remdesivir
+             * Casirivimab and imdevimab
+        """,
+    )
+    load_date = Series(
+        datetime.date,
+        description="Date on which the current dataset was imported.",
+    )
+    received = Series(
+        datetime.date,
+        description="Date form submitted.",
+    )
+    region = Series(
+        str,
+        description="""
+            NHS England region in which the CMDU submitting the form is located.
+        """,
+    )
+    risk_cohort = Series(
+        str,
+        description="""
+            High-risk group to which the patient was considered to belong. Derived from
+            tick-boxes. Multiple groups can be selected and will be comma separated,
+            e.g. `liver disease,rare neurological conditions`.
+
+            This is an aggregation of `CASIM05_risk_cohort`, `MOL1_high_risk_cohort`
+            and `SOT02_risk_cohorts`.
+
+            The available groups as at the time of writing are listed below. However
+            note that the precise wording used has changed over time and so filtering by
+            a specific disease name may not be reliable.
+
+             * `Downs syndrome`
```

Comment:
Check this list against the latest one (see therapeutic-risk-groups workspace in Airlock)

Reply:
Just checked this list against those lists in Airlock & they match (with the caveat that as discussed elsewhere there are ` and `-separated combinations which we convert to comma-separated combinations).